DigitalLearningSolutions.Web/Controllers/FindYourCentreController.cs

```diff
@@ -11,9 +11,8 @@
     using DigitalLearningSolutions.Web.Models.Enums;
     using DigitalLearningSolutions.Web.ViewModels.FindYourCentre;
     using Microsoft.AspNetCore.Mvc;
-    using Microsoft.FeatureManagement.Mvc;
+    using Microsoft.Extensions.Configuration;
 
-    [FeatureGate(FeatureFlags.RefactoredFindYourCentrePage)]

```

Comment:
Removed this so it would serve up the old functionality if the feature was toggled off. In hindsight this isn't ideal. Is there preferred method to serving different indexes depending on a feature on the same controller or is it a case of creating separate controllers for the old and new functionality?

Reply:
Check the flag in the Index action instead and just return a different view? Probably slightly better than the in view stuff you've done.